src/Plugin.php

```diff
@@ -129,4 +150,27 @@ public function process(Event $event): void
 		}
 	}
 
+	private function getInstallPath(InstallationManager $installationManager, PackageInterface $package): string
```

Comment:
An alternative solution could be to rely on `Composer\Util\Filesystem::findShortestPathCode` instead (but it might require changing the way `$data` is handled for the file generation then as `var_export($data, true)` won't do it anymore)